From b5e17a613d2ea9894fcc090499dcc73e3ea07f61 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Mon, 9 Aug 2021 15:25:50 +0100
Subject: [PATCH 27/30] tests: add aarch64 and mips64 as a valid 64-bit
 machines

require_64_bit_ in t-lib-helpers.sh has a hard-coded list of uname
machines that are 64-bit, so add aarch64 and mips64 to cover the major
architectures.

Signed-off-by: Brian C. Lane <bcl@redhat.com>
---
 tests/t-lib-helpers.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/t-lib-helpers.sh b/tests/t-lib-helpers.sh
index 9312343..dddb44e 100644
--- a/tests/t-lib-helpers.sh
+++ b/tests/t-lib-helpers.sh
@@ -411,7 +411,7 @@ device_mapper_required_()
 require_64bit_()
 {
   case $(uname -m) in
-      x86_64|ppc64)
+      aarch64|mips64|ppc64|x86_64)
           return 0;;
       *)
           skip_ "This test requires a 64 bit system"
-- 
2.31.1

